cmd/commands/cli/command.go

```diff
@@ -159,20 +161,111 @@ func (c *cliApp) handleActions(line string) {
 	}
 }
 
+func (c *cliApp) service(argsString string) {
+	printHelp := func() {
+		fmt.Println("service <action> [args]")
+		fmt.Println("	start	<ProviderID> <ServiceType> [options]")
+		fmt.Println("	stop	<ServiceID>")
+		fmt.Println("	list")
+		fmt.Println("	get	<ServiceID>")
+		fmt.Println("")
+		fmt.Println(`example: service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn {"port":1190, "protocol":"UDP"}`)
```

Comment:
Are we sure we want to pass options this way? Such approach might not be good for scripting and command line usage. Why not use a common way?

```suggestion
		fmt.Println(`example: service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn 
 --port=1190 --protocol=UDP`)
```

Reply:
I like how iproute2 utility organizes its commands / arguments. In that case we wouldn't even need '--'s just: 
` service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn port 1190 protocol UDP`

Reply:
and all options should be optional

Reply:
All options are optional and have the same representation as for service subcommand now.